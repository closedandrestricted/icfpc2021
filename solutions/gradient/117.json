{"vertices":[[146,75],[148,59],[130,40],[123,73],[130,77],[124,80],[103,86],[110,46],[123,93],[111,56],[120,52],[111,77],[106,89],[119,71],[102,84],[107,109],[34,91],[94,85],[54,64],[67,57],[57,58],[87,88],[59,48],[60,60],[63,66],[47,85],[90,94],[110,90],[65,97],[70,61],[86,55],[49,35],[55,44],[79,59],[62,92],[94,110],[84,112],[103,119],[59,121],[110,119],[123,122],[94,132],[120,120],[119,127],[110,141],[135,129],[78,120],[84,128],[115,128],[80,139],[151,139],[109,113],[95,126],[127,110],[136,106]]}