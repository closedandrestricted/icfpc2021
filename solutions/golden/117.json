{"vertices":[[142,80],[145,64],[129,42],[124,83],[129,89],[123,93],[119,113],[110,46],[110,88],[111,56],[119,54],[111,82],[98,81],[116,75],[100,87],[96,109],[33,91],[91,89],[73,64],[86,53],[78,59],[84,92],[60,51],[62,63],[65,69],[47,87],[91,93],[112,93],[62,101],[71,63],[70,52],[51,38],[57,47],[65,56],[61,96],[92,109],[86,116],[103,113],[61,125],[109,116],[121,121],[93,133],[117,120],[116,127],[109,142],[133,125],[76,124],[82,132],[113,128],[77,143],[147,138],[104,115],[91,129],[122,112],[131,108]]}